nixos/tests/seafile.nix

```
@@ -0,0 +1,120 @@
+import ./make-test-python.nix ({ pkgs, ... }:
+  let
+    client = { config, pkgs, ... }: {
+      virtualisation.memorySize = 256;
+      environment.systemPackages = [ pkgs.seafile-shared ];
```

small seahub smoke test, part 1

```suggestion
      environment.systemPackages = [ pkgs.seafile-shared curl ];
```